revamp README (#13)
SSAGESLabs/hoomd-dlext: README.md

# HOOMD-dlext

Provide access to [HOOMD-blue](https://hoomd-blue.readthedocs.io/en/v2.9.7/) simulation data on CPU or GPU to python via [DLPack](https://github.com/dmlc/dlpack)
This project is primarily designed to enable access to HOOMD-blue for the [PySAGES](https://pysages.readthedocs.io/en/latest/) project.
At the moment, we only support HOOMD-blue version 2.
Support HOOMD-blue version 3 is planned and will be released later.

## Installation

Follow the [Plugins and Components Guide](https://hoomd-blue.readthedocs.io/en/v2.9.7/developer.html) from the HOOMD-blue reference documentation site.
For system requirements, check HOOMD-blue's install [requirements](https://hoomd-blue.readthedocs.io/en/v2.9.7/installation.html#compiling-from-source).
At the moment we only support installations on Linux and Mac.
For GPU support, the base HOOMD-blue installation must be installed for CUDA GPUs.

> For example installation as an external component can be performed after HOOMD-blue is already installed on the system.

Reviewer: ```suggestion
Assuming HOOMD-blue is already installed on the system, the plugin can be installed as an external component
```

> The next step is to obtain a copy of this plugin preferably via `git clone`.

Reviewer: ```suggestion
First, we obtain a copy of this plugin, for example via `git clone`.
```

> ```shell
> cd /path/to/hoomd-dlext
> ```

Reviewer: ```suggestion
```shell
git clone https://github.com/SSAGESLabs/hoomd-dlext.git
cd hoomd-dlext
```
```

> Now we configure the installation with CMake. Here it is important, that the python version detected by CMake can successfully `import hoomd`.

Reviewer: ```suggestion
We then configure the installation with CMake. It is important, that the python version detected by CMake can successfully `import hoomd`.
```

```shell
mkdir build && cd build
cmake ..
```

> And finally we compile the plugin on your target machine

Reviewer: ```suggestion
And finally we compile the plugin on the target machine
```

```shell
make
```

> and install it into the HOOMD-blue installation.
> It is important to have a writeable installation for this to work.

Reviewer: ```suggestion
and install it into the HOOMD-blue installation path (the latter must be writeable for this to work).
```
